airbyte-integrations/connectors/destination-jdbc/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
 
     implementation project(':airbyte-db:lib')
     implementation project(':airbyte-integrations:bases:base-java')
+    implementation project(':airbyte-integrations:connectors:destination-s3')
```

Comment:
> I slightly tried doing that earlier today and ended up pulling waaaay too many files into base-s3, but will give this another attempt.

Got it. Right, the s3 stream copier probably should be in the jdbc module, since some of the jdbc destination uses a staging s3.

Looks like this is tricky to do. Feel free to merge the PR as is.

Reply:
yep - trying to move S3DestinationConfig pulls in S3FormatConfigs -> S3ParquetFormatConfig(/avro/jsonl/csv) -> various parquet(/avro) libraries; leaving this as-is

Reply:
Got it. All good then.